### PR TITLE
refactor test_issue_142 and test_issue_143

### PR DESCRIPTION
**Pull request by jacadzaca**
splitting as per #400

### CODE REVIEW

File: src/icalendar/tests/test_parsing.py [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+from icalendar.parser import Contentline, Parameters
+
+@pytest.mark.parametrize('raw_content_line, expected_output', [
+    # Issue #142 - Multivalued parameters. This is needed for VCard 3.0.
+    # see https://github.com/collective/icalendar/pull/142
+    ('TEL;TYPE=HOME,VOICE:000000000', ('TEL', Parameters({'TYPE': ['HOME', 'VOICE']}), '000000000')),
+    # Issue #143 - Allow dots in property names. Another vCard related issue.
+    # see https://github.com/collective/icalendar/pull/143
+    ('ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR:;;This is the Adress 08; Some City;;12345;Germany', ('ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR', Parameters(), ';;This is the Adress 08; Some City;;12345;Germany'))
```
Comment (angatha) [inline src/icalendar/tests/test_parsing.py]: This line is quite long.

Comment (angatha) [inline src/icalendar/tests/test_parsing.py]: Is there a missing test for folded input e.g. `'ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR:;;This is the Adress 08; \r\n Some City;;12345;Germany'`

File: src/icalendar/tests/test_fixed_issues.py [hunk 2]
```diff
@@ -268,43 +268,6 @@ def test_issue_116(self):
...
-
-        ctl = Contentline.from_ical("ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR:;;This is the Adress 08; Some City;;12345;Germany")  # nopep8
-        self.assertEqual(
-            ctl.parts(),
-            ('ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR',
-             Parameters(),
-             ';;This is the Adress 08; Some City;;12345;Germany'),
-        )
-
-        ctl2 = Contentline.from_ical("ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.X-ABLABEL:")  # nopep8
```
Comment (angatha) [inline src/icalendar/tests/test_fixed_issues.py:111]: This test case was dropped. Can you please add an equivalent test case or was that not required behavior?

**Review by angatha — APPROVED** [review]

File: src/icalendar/tests/test_fixed_issues.py (hunk 2 above)
Comment (niccokunzmann) [reply src/icalendar/tests/test_fixed_issues.py:111]: It is here:
https://github.com/collective/icalendar/pull/416/files#diff-8b1f574fcce32e4b0c21919811f7aed7f9346116f893c629847faece2368b46bR16

**Review by niccokunzmann — APPROVED** [review]
I see that everything is transferred to the new structure. Thanks!

File: src/icalendar/tests/test_parsing.py (hunk 1 above)
Comment (niccokunzmann) [reply src/icalendar/tests/test_parsing.py]: Is this an addition? If yes, please create an issue or a PR to solve this. :heart:

**Pull request merged**